### ISSUE TITLE
Script Builder Broken on 3.12.13

### ISSUE THREAD

**Issue by jvrsgsty** [SublimeText/LaTeXTools#1000]
Everything was working perfectly and after the update, building using the script builder fails. 
I am working on OSX and ST2. 

My working script-builder setup from several months ago (extract from my settings file) :
```
	"builder": "script",
	"builder_path": "",
	"builder_settings" : {
		"display_log" : false,	
		"osx": {
			"script_commands": [
				"pdflatex -synctex=1 -interaction=nonstopmode",
				"bibtex",
				"pdflatex -synctex=1 -interaction=nonstopmode",
				"pdflatex -synctex=1 -interaction=nonstopmode"
			]
		},
		"windows" : {
			// See README or third-party documentation
		},
		"linux" : {
			// See README or third-party documentation
		}
	}
```
Console output:
```
theme loaded
Loaded LaTeXTools plugins [] from path /Users/jvrsgsty/Library/Application Support/Sublime Text 2/Packages/LaTeXTools/bibliography_pluginsapp ready

pre session restore time: 1.51865
using gamma: 2 (err: 6.9282)
wrote startup cache, added files: 11 orphaned files: 0 total files: 109 cache hits: 98
startup time: 1.77353 (package setup was not run)
found 5 files for base name Main.sublime-menu
Loaded LaTeXTools plugins [] from path /Users/jvrsgsty/Library/Application Support/Sublime Text 2/Packages/LaTeXTools/builders/pdfBuilder.py
Generating syntax summary
Loaded LaTeXTools plugins [] from path /Users/jvrsgsty/Library/Application Support/Sublime Text 2/Packages/LaTeXTools/builders
loaded 1000 snippets
<class '_latextools_scriptBuilder.ScriptBuilder'>
Welcome to thread Thread-7
Running "p d f l a t e x ' ' - s y n c t e x = 1 ' ' - i n t e r a c t i o n = n o n s t o p m o d e ' ' S o P"
4
Traceback (most recent call last):
  File "./makePDF.py", line 317, in run
AttributeError: 'NoneType' object has no attribute 'decode'
```
It seems the builder is somehow inserting spaces between each character in the commands. Running the commands on the script-builder config in an OS terminal directly works perfectly.

**Comment by ig0774** [comment]
Thanks for pointing this out. It will be fixed in the next release. For now, you can update the script builder to [this version](https://raw.githubusercontent.com/SublimeText/LaTeXTools/master/builders/scriptBuilder.py) on your local copy and things should work.

**Comment by jvrsgsty** [comment]
Fixed on 3.12.15, at least. Might also be fixed on 3.12.14, but didn't try that out. 
Great! 🎊
